src/snake.py

```diff
@@ -1,23 +1,54 @@
+"""Snake Module"""
+from typing import List
+from src.coordinate import Coordinate
+
 class Snake:
-    def __init__(self, data):
+    """Define properties and functionality of a Snake."""
+    def __init__(self, data: dict):
         self._data = data
+        self._coordinates: List[Coordinate] = []
+
+    def __eq__(self, other: object) -> bool:
+        """Return true if the snake id, health, and coordinates match."""
+        if not isinstance(other, Snake):
+            return NotImplemented
+        return self.id == other.id
+
+    def __str__(self) -> str:
+        return str(self.id) + "; " + str(self.health)# + "; " + str(self.head.x) +","+ str(self.head.y)
 
     @property
-    def id(self):
-        return self._data['id']
+    def id(self) -> str:
+        """Get the id of the snake."""
+        return self._data["id"]
 
     @property
-    def coordinates(self):
-        return self._data['coords']
+    def coordinates(self) -> List[Coordinate]:
+        """Get the List of Coordinates which describes the location of this Snake."""
+        if self._coordinates is not None and len(self._coordinates) > 0:
```

Comment:
inconsistent to how you did it in Board, see my comments there for reference.

- [ ] Refactor to match style in board